native/NativeSampleApps/RestExplorer/src/com/salesforce/samples/restexplorer/RestExplorerApp.kt

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (c) 2011-present, salesforce.com, inc.
```

Comment:
NIT should the range be 2024-present since it's a new file?

Reply:
That's a neat question since this is a rename due to the Kotlin conversion.  Logically, it's still the same RestExplorerApp class.  From a historical perspective, I do kind of like the hint that this has git history going back before the rename, which is something I've used quite a bit in researching Kotlin logic in its Java form to learn requirements.